docs/projects/ad7606x_fmc/index.rst

```diff
@@ -0,0 +1,438 @@
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+   * - adc_serpar
+     - INOUT
+     - 39
+     - 93
+   * - adc_refsel *
+     - OUT
+     - 38
+     - 92
+   * - adc_first_data **
+     - INOUT
+     - 38
+     - 92
+   * - adc_reset
+     - INOUT
+     - 37
+     - 91
+   * - adc_stby
+     - INOUT
+     - 36
+     - 90
+   * - adc_range
+     - INOUT
+     - 35
+     - 89
+   * - adc_os
+     - INOUT
+     - 34:32
+     - 88-86
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+=============== === ========== ===========  
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_ad7606_dma  13  57         89
+spi_ad7606 **   12  56         88
+=============== === ========== ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <main:/>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/ad7606x_fmc/zed
+   user@analog:~/hdl/projects/ad7606x_fmc/zed$ make DEV_CONFIG=2 INTF=0
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make DEV_CONFIG=2 INTF=0``
+
+then the folder name will be:
+
+``DEVCONFIG2_INTF0``
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Connections and hardware changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   **The following hardware changes are required:**
+
+   Depending on the required interface mode, some hardware modifications need to
+   be done.
+
+   -   **JP5** - A - Serial interface
+   -   **JP5** - B - Parallel interface
+   -   **JP7, JP12, JP13, JP14** - B - Differential operation
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`AD7606X HDL Reference Design <resources/eval/user-guides/ad7606x-fmc/hdl>`
```

Comment:
If there is a quick start guide, mention it in here in a table format like this: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad9783_ebz/index.rst?plain=1#L143

otherwise, delete the entire  "Systems related" section.